ONSdigital/census-rh-ui
# Cr 1499 fulfilment session invalidate

# Motivation and Context
Invalidates the user session when a fulfilment is successfully completed

# What has changed
Added the 'invalidate' call to the render of the 'success' pages in all fulfilment journeys to kill off the users session. This prevents the session continuing if they wish to make further requests

No Cucumber updates required

# How to test?
Make a fulfilment request, and ensure that the user session is removed from the browser when reaching the 'success' screen

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1499

# Screenshots (if appropriate):
